### PR TITLE
Add support for 64-bit addresses and values

### PR DESCRIPTION
**Pull request by Disasm**

### CODE REVIEW

**Comment by Disasm** [comment]
Note that the additional changes I made are mostly "fix everything that doesn't compile anymore" and I could have missed something.

File: src/svd/peripheral.rs [hunk 1]
```diff
@@ -76,7 +76,7 @@ pub struct Peripheral {
 #[derive(Clone, Debug, Default, PartialEq)]
 pub struct PeripheralBuilder {
```
Comment (burrbull) [inline src/svd/peripheral.rs:77]: Maybe it makes sense to parametrize structures with size: `PeripheralBuilder<U>` instead of replacing all u32 with u64?

Comment (Disasm) [reply src/svd/peripheral.rs:77]: Hmm, looks like unnecessary complication, given that the size type is not something the user can change.

**Comment by Disasm** [comment]
Rebased to fix conflicts and added support for 64-bit writeConstraint and enum values.

**Review by burrbull — APPROVED** [review]

**Review by therealprof — APPROVED** [review]
LGTM, too. Thanks.

bors r+

**Comment by therealprof** [comment]
Oh crap... This is still on Travis CI. 🙄

**Pull request merged**